prestart.ts

```diff
@@ -49,7 +49,7 @@ const waitForTunnel = async () => {
 const callQueues = async () => {
 	const queueUrls = Object.entries(process.env)
 		.filter(([key, value]) => /^SQS_.*_QUEUE_URL$/.test(key) && value)
-		.map(([_key, value]) => value as string);
+		.map(([_key, value]) => `${value}?Action=GetQueueUrl&QueueName=${value!.split("/").pop()}`);
```

Comment:
Have to update this based on the AWS API to make sure queue exists on localstack